### ISSUE TITLE
Incorrect label for the fullscreen toolbar button

### ISSUE THREAD

**Issue by Comandeer** [ckeditor/ckeditor5#18335]
## 📝 Provide detailed reproduction steps (if any)

1. Open a sample with a fullscreen plugin.
2. Check the label of the fullscreen menu button.
3. Click it to enter the fullscreen mode.
4. Check the label of the button.

### ✔️ Expected result

The label should stay the same.

### ❌ Actual result

Label changes from "Enter fullscreeen mode" to "Leave fullscreen mode".

![The "Enter fullscreen mode" label](https://github.com/user-attachments/assets/092f5060-0ecf-4b8a-85d0-ba0c12259deb)
![The "Leave fullscree mode" label](https://github.com/user-attachments/assets/c46aee3c-4c31-486e-8873-13360c8ea03f)

This behavior is inconsistent with the behavior of any other toggle button (e.g. we have “Bold” with on/off state, not “Bold” and “Unbold”; also it’s “Bold”, not “Start bolding”).

### ❓ Possible solution

The label should be changed to "Fullscreen mode" – the state of the button is already expressed by its styles and `[aria-pressed]` attribute.

## 📃 Other details

* Browser: Chrome 134.0.6998.166 
* OS: macOS 15.4
* First affected CKEditor version: `master`
* Installed CKEditor plugins: `fullscreen`

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
